lib/adrian/directory_queue.rb

```diff
@@ -69,9 +72,10 @@ def reserve(item)
     end
 
     def items
-      items = files.map { |file| wrap_item(file) }
-      items.reject! { |item| !item.exist? || filter?(item) }
-      items.sort_by(&:updated_at)
+      return @items if @items && @items.length > 0
+      @items = files.map! { |file| wrap_item(file) }
+      @items.reject! { |item| !item.exist? || filter?(item) }
```

Comment:
Not super easily because filter? can be used on a list of QueueItems rather than FileItems and QueueItem does not have an exist? method. We know we have FileItems here because we've "wrapped" them.